resolver/resolver.cc

```diff
@@ -425,6 +425,8 @@ class ResolveConstantsWalk {
             } else {
                 if (auto e = ctx.state.beginError(job.ancestor->loc, core::errors::Resolver::RedefinitionOfParents)) {
                     e.setHeader("Class parents redefined for class `{}`", job.klass.data(ctx)->show(ctx));
+                    e.addErrorLine(job.ancestor->loc, "Parent redefined from `{}` to `{}`",
+                                   job.klass.data(ctx)->superClass().show(ctx), resolved.show(ctx));
```

Comment:
This is tricky. The loc we'd really like to show is the previous class def ancestor. I'm not sure if we have that loc.

It was kind of confusing to me to see the same line in the error header and error line... Do you think there's anything we can do to fix that? I wonder if just moving the `from ... to` into the error header and dropping the error line would help.

Up to you.

Reply:
I was worried about the line length in practice (e.g. if all the class names are fully qualified, five-level-deep names) but it's probably clearer to put it all on one line.